src/style.css

```diff
@@ -0,0 +1,223 @@
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@300;400;500;600&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Source Sans Pro', sans-serif;
+}
+
+li {
+  list-style: none;
+}
+
+body {
+  width: 100%;
+  height: 100vh;
+  padding: 10px;
+  background-color: #eee;
+}
+
+.todo-card {
+  max-width: 480px;
+  width: 100%;
+  margin: 120px auto;
+  border-radius: 8px;
+  border: 1px solid rgba(0, 0, 0, 0.24);
+  background-color: whitesmoke;
+}
+
+.todo-card header {
+  font-size: 1.3rem;
+  font-weight: 600;
+  padding: 12px;
+  color: rgba(0, 0, 0, 0.67);
+  border-bottom: 1px solid rgba(0, 0, 0, 0.24);
+}
+
+.list-item {
+  display: flex;
+  justify-content: space-between;
+  font-size: 1.1rem;
+  font-weight: 500;

```

Comment:
- [OPTIONAL]  I noticed that these two properties are shared between four elements. Please consider making them page defaults by moving them in the body properties.